### PR TITLE
Update add_tool.md

### PR DESCRIPTION
**Pull request by civier**
1) changed neurodesktop-dev to neurodesktop
2) request people to tell us after they tested their app in the daily build. We will announce the new release only then
3) if people want to use the new app before the next release is out, I told them that they can install neurodesktop-latest (I hope it is ok. If you want to limit neurodesktop-latest for developers only, let me know why, and we can discuss)

### CODE REVIEW

File: content/en/developers/new_tools/add_tool.md
Comment (stebo85) [inline content/en/developers/new_tools/add_tool.md:162]: typo: Post an issue in the open an issue ->

File: content/en/developers/new_tools/add_tool.md
Comment (stebo85) [inline content/en/developers/new_tools/add_tool.md:163]: The windows/mac/linux startup binaries will also start the latest version automatically

**Review by stebo85 — CHANGES_REQUESTED** [review]

File: content/en/developers/new_tools/add_tool.md
Comment (civier) [reply content/en/developers/new_tools/add_tool.md:163]: Is that what we want? Don't we want the developer who built the new container to test the final version before people are using it? Let's decide on it, and then the Swinburne team can make the required changes to the binaries (unless we are not familiar with the tools used to develop them).

File: content/en/developers/new_tools/add_tool.md
Comment (civier) [reply content/en/developers/new_tools/add_tool.md]: I committed a correction

File: content/en/developers/new_tools/add_tool.md
Comment (stebo85) [reply content/en/developers/new_tools/add_tool.md:163]: the check of the tool as to happen before it's merged into the apps.json file - after that it's already live on CVMFS

**Review by stebo85 — APPROVED** [review]

File: content/en/developers/new_tools/add_tool.md
Comment (civier) [reply content/en/developers/new_tools/add_tool.md:163]: I know. But sometimes there are problems in the launch through the menu. Isn't it? (environment variables, folder binding,  etc.) Correct me if wrong.

Comment (stebo85) [reply content/en/developers/new_tools/add_tool.md:163]: shouldn't - but there is no other way than building the image and testing it

Comment (civier) [reply content/en/developers/new_tools/add_tool.md:163]: I'm not familiar with how neurodesktop:latest is built exactly, but I'll be wary of letting end users use a version that neither of the developers at least tried once. There can always be problems we didn't think of, even in parts other than the menu.

I say to keep the recommendation to use the last release version, and update the launch binaries accordingly.

If you think differently, we can discuss it in our next meeting.

Comment (stebo85) [reply content/en/developers/new_tools/add_tool.md:163]: I don't mind. I am happy with the launch binaries extracting the latest tested release from our website (but this is currently not implemented) and starting it. Currently the launch binaries start latest, as this was the quickest way to get it working.

Comment (civier) [reply content/en/developers/new_tools/add_tool.md:163]: OK. I noted it down and check if we have the expertise to modify the binaries in Swinburne. If we can do it, I'll update you with the estimated deadline.

Comment (stebo85) [reply content/en/developers/new_tools/add_tool.md:163]: great! It is straight forward for Linux and Mac, but not for powershell. The latest tested version needs to be extracted from here:
 https://raw.githubusercontent.com/NeuroDesk/neurodesk.github.io/hugo-docsy/data/neurodesktop.toml

Comment (civier) [reply content/en/developers/new_tools/add_tool.md:163]: Hi @stebo85
Just want to update that I was busy with building containers and now will be busy with the talk, so unfortunately, we'll need to put the extraction of last tested version on hold. I don't think that we have an actual issue for that, but in case we do, you're welcome to mark it as "won't fix right now". Sorry for that, but there are indeed higher priorities.

**Pull request merged**
